content/docs/_index.md

```diff
@@ -6,7 +6,7 @@ lead: "Learn all about how the atPlatform works" # The lead below the title (ON
 
 description:
   | # SEO Description of the page (Shows in google and atsign.dev search)
-  Documentation for the atPlatform.
+  Documentation for the atPlatform of atSign Foundation. Learn how to use our tools and SDK!
```

Comment:
Please limit your PR to one set of changes.

Reply:
Well, the initial task was also to update the description in some... more prettier way, so, I decided to change that too

Reply:
Yes @XavierChanth I asked for a better description in the discord embed in the issue card #137 

Does this change reflect the discord embed @daniloff200 ?

Reply:
Discord uses [opengraph](https://ogp.me/) for their embeds, [see here](https://gohugo.io/templates/internal/#configure-open-graph) for how to implement the metadata in Hugo.

Reply:
hi, @XavierChanth . Yeah, I've also seen that doc. I should say, it's not completely working for this case, because, each .md file has its own description & title, which overrides the og tags

Reply:
I should remove that, I think, because, we need to change bunch of titles, if we want to rework them, not only this one

Reply:
> hi, @XavierChanth . Yeah, I've also seen that doc. I should say, it's not completely working for this case, because, each .md file has its own description & title, which overrides the og tags

Are you sure that it overrides the og tags? As far as I know these are different from the basic meta tags (which are modified on a per-page basis).

Reply:
I can show u :) 
On my video you can see, that some changes in config was not set, but, changes in title / description take place not only on the page, but, also for OG tags
[Untitled_ Oct 5 2022 10_24 PM.webm](https://user-images.githubusercontent.com/13692220/194145233-be90bae9-fa59-4d4f-a706-3f083ffea4fe.webm)

Reply:
Thanks for the insight, that was not the behavior I expected, but after thinking about it, it makes sense. I will resolve and approve this, and open a new issue to update the descriptions across the various pages.

Reply:
Thanks! Yeah, that was also kinda unexpectable for me, but, it is what it is in our work. Every day you learn something new, sometimes some strange stuff ;)